01-messaging.md

```diff
@@ -135,7 +135,8 @@ according to the message-specific format determined by `type`.
 ### Requirements
 
 The sending node:
- - MUST order `tlv_record`s in a `tlv_stream` by monotonically-increasing `type`.
+ - MUST order `tlv_record`s in a `tlv_stream` by strictly-increasing `type`,
```

Comment:
strictly-increasing is not the same as "unique", which this sentence implies with "i.e."

Reply:
Strictly increasing does imply unique, doesn't it? (the reverse isn't true)

Reply:
Below we have: 
> The strict monotonicity constraint ensures that all `type`s are unique and can at most once. Fields that map to complex objects, e.g. vectors, maps, or	appear at most once

Reply:
indeed, but "i.e." implies that they are equivalent in that, as merriam-webster says "it introduces a rewording or a clarification". An implication is not a rewording.

Reply:
Is your suggestion to change `i.e. MUST not produce` to `hence MUST not produce`?
If so, @dr-orlovsky can you apply this small change and we can merge?

Reply:
yes, that is a great idea.

Reply:
done!